src/js/extensions/paste.js

```diff
@@ -155,31 +154,6 @@ var PasteHandler;
             elList = text.split('<br><br>');
 
             this.pasteHTML('<p>' + elList.join('</p><p>') + '</p>');
-
-            try {
-                this.document.execCommand('insertText', false, '\n');
```

Comment:
The only thing I can think of why we might do this:

1. Something with undo/redo maybe?  Did you check to see how undo/redo worked with pasting before/after you removed this line?
1. If you paste at the end of chunk of text, does something weird happen?  Maybe the extra newline gave users a place to put their cursor so they can add more text without inheriting the formatting from the pasted text?